chain/default_store.go

```diff
@@ -113,10 +114,15 @@ func (store *DefaultStore) Load(ctx context.Context) error {
 		if err != nil {
 			return errors.Wrap(err, "failed to add validated block to TipSet")
 		}
+		startHeight = blk.Height
 	}
+	logStore.Infof("start loading chain at tipset: %s, height: %d", tipCids.String(), startHeight)
 
 	var genesii types.TipSet
 	err = store.walkChain(ctx, headTs.ToSlice(), func(tips []*types.Block) (cont bool, err error) {
+		if (tips[0].Height % 100) == 0 {
+			logStore.Infof("load tipset: %s, height: %v", tips[0].Cid().String(), tips[0].Height)
```

Comment:
Not sure if we want to emit a log every-so-often about the current state of loading the chain, could be removed, or a debug message. Thoughts?

Reply:
Rather than log every N blocks, how about every `startHeight/M` so that no more than M messages are logged, even when the chain is hundreds of thousands of blocks long?

Reply:
isn't this the point of the pr?

Reply:
wasn't sure, issues started we should have a message when we start and when we finish, consensus is this appears to be useful so we'll leave it in